packages/app-page-builder/src/editor/plugins/elements/heading/elementText.ts

```diff
@@ -17,13 +17,11 @@ const lexicalText = JSON.stringify({
                         version: 1
                     }
                 ],
-                styleId: "heading1",
-                name: "Heading 1",
                 direction: "ltr",
-                typographyStyles: {},
+                styles: [],
                 format: "",
                 indent: 0,
-                type: "typography-el-node",
+                type: "base-heading-node",
```

Comment:
On the back of my comments re BaseParagraph naming... can we use `heading-node` here instead of using the word `base` ? I'd like to avoid `base` since that could imply it's a base for something, which it isn't. It's just a node.